### PR TITLE
Add customOnScroll as Animated.event listener

### PR DESCRIPTION
**Pull request by Definitely-Not-Vlad**

### CODE REVIEW

File: src/drivers/ScrollDriver.js [hunk 1]
```diff
@@ -26,10 +26,12 @@ import DriverBase from './DriverBase';
  * @param options.nativeScrollEventThrottle Native animated value changes
  *   will be debounced using this value when mirroring them to the JS value.
  *   Used only if `useNativeDriver` is `true`, defaults to 20ms.
+ * @param customOnScroll Function to be executed in tandem with ours.
  */
 export default class ScrollDriver extends DriverBase {
   constructor(
     options = { useNativeDriver: false, nativeScrollEventThrottle: 20 },
```
Comment (sstimac) [inline src/drivers/ScrollDriver.js:33]: we're not sending ( overriding the default value ) for the nativeScrollEventThrottle from our scrollview, as we're only sending the useNativeDriver prop. I think this is a bug. Please check and include inside our ScrollView if i'm right ( i know it's not related to the bug, but let's sort it out while we're at it )

**Review by sstimac — CHANGES_REQUESTED** [review]
just naming and something i noticed. Very nice otherwise

File: src/drivers/ScrollDriver.js (hunk 1 above)
Comment (Definitely-Not-Vlad) [reply src/drivers/ScrollDriver.js:33]: Why would I change this behavior? I'm not sure what the bug is that you're mentioning.

If we set `useNativeDriver` to `true`, but don't set `nativeScrollEventThrottle`, it defaults to `20`. What's the bug?

Comment (Definitely-Not-Vlad) [reply src/drivers/ScrollDriver.js:33]: Also you mention naming in the main comment of the CR, but I don't see any comments regarding naming, could you add them if there are any please

File: src/drivers/ScrollDriver.js [hunk 2]
```diff
@@ -26,10 +26,12 @@ import DriverBase from './DriverBase';
...
+    customOnScroll,
```
Comment (sstimac) [inline src/drivers/ScrollDriver.js:34]: i'd like this to be called just `onScroll`

**Review by sstimac — CHANGES_REQUESTED** [review]
pls check my comments

**Review by sstimac — APPROVED** [review]

**Pull request merged**